src/Components/Reviews/FactorsComponents/WidthBarComponent.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+const WidthBar = ({ width }) => {
+  if (!width) {
+    return null;
+  }
+  const { value } = width;
+  const roughPercent = value / 5;
+  const roundedPercent = `${Math.round(roughPercent * 100)}%`;
```

Comment:
This component seems very similar to the size bar above it may be worth it to modularize the SVG components if you are planning on building more